static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import * as segmentUtil from 'common/modules/experiments/segment-util';
+import * as testCanRunChecks
+    from 'common/modules/experiments/test-can-run-checks';
+import * as viewLog from 'common/modules/commercial/acquisitions-view-log';
```

Comment:
How do you feel about destructuring these imports instead of `splat`ting? For the tree-shaking (( 🌴 ))